pat/ReferenceTestV2/consumer/step_5.py

```diff
@@ -0,0 +1,144 @@
+            step=step,
+            message=f'Alert condition cause-remedy information updates are more than {max_time_between_updates} '
+            f'seconds apart: {[update for update in updates if update > max_time_between_updates]}',
+        )
+
+    updates: list[float] = mdib.xtra.unit_of_measure_updates
+    if max(updates) <= max_time_between_updates:
+        result_collector.ResultCollector.log_success(
+            step=step,
+            message=f'Unit of measure updates are at least {max_time_between_updates} seconds apart',
+        )
+    else:
+        result_collector.ResultCollector.log_failure(
+            step=step,
+            message=f'Unit of measure updates are more than {max_time_between_updates} seconds apart: '
+            f'{[update for update in updates if update > max_time_between_updates]}',
+        )
+
+
+def _on_description_modification_report(
+    step: str,
+    created_vmds: set[str],
+    start_waiting_event: threading.Event,
+    delete_after_create_event: threading.Event,
+    dmr: msg_types.DescriptionModificationReport,
+):
+    for report_part in dmr.ReportPart:
+        report_part: msg_types.DescriptionModificationReportPart
+        logger.debug(
+            'received description modification report part %s with descriptor handles %s',
+            report_part.ModificationType,
+            [descriptor.Handle for descriptor in report_part.Descriptor],
+            extra={'step': step},
+        )
+        for descriptor in report_part.Descriptor:
+            if pm_qnames.VmdDescriptor != descriptor.NODETYPE:
+                continue
+            if report_part.ModificationType == msg_types.DescriptionModificationType.CREATE:
+                if descriptor.Handle in created_vmds:
+                    logger.error(
+                        'Descriptor already created with the handle "%s"',
+                        descriptor.Handle,
+                        extra={'step': step},
+                    )
+                else:
+                    created_vmds.add(descriptor.Handle)
+                    start_waiting_event.set()
+            if report_part.ModificationType == msg_types.DescriptionModificationType.DELETE:
+                if descriptor.Handle in created_vmds:
+                    logger.info(
+                        'VMD with the handle %s deleted after creation',
+                        descriptor.Handle,
+                        extra={'step': step},
+                    )
+                    delete_after_create_event.set()
+
+
+def test_5b(mdib: ConsumerMdib):
+    """The Reference Provider produces at least 1 insertion followed by a deletion every 10 seconds comprising
+
+    - Insert a VMD including Channels including metrics (inserted VMDs/Channels/Metrics are required to have a new handle assigned on each insertion such that containment tree entries are not recycled). (Tests for the handling of re-insertion of previously inserted objects should be tested additionally)
```

Comment:
is it even needed to test that at least one channel and at least one metric is inserted? current implementation is not testing this